peribolos/knative-sandbox.yaml

```diff
@@ -351,7 +359,10 @@ orgs:
           general project administration.
         privacy: closed
         repos:
+          # All repos should be listed here!
```

Comment:
As an FYI, being org admins should make this redundant.

Reply:
I do not think that the Prow bots nor the release leads are org admins